### ISSUE TITLE
demo code gives error

### ISSUE THREAD

**Issue by setcooki** [lepture/github-cards#64]
when i embed the code example somewhere i get

Couldn't find the requested file /cards/default.html in lepture/github-cards.

**Comment by khalby786** [comment]
Yeah, I've been encountering this as well. I think it's something wrong with JSDelivr.

**Comment by AftDawn** [comment]
it seems to be 404ing on a GET request, that's most likely the reason why it's not working

**Comment by khalby786** [comment]
Considering how this repo's been dead for a year now, I'm not sure if this will get fixed.
